adam/language_specific/english/english_language_generator.py

```diff
@@ -606,8 +607,37 @@ def _preposition_for_region_as_goal(self, region: SituationRegion) -> str:
                 # TODO: put constraints on the axis
             ):
                 return "on"
+            elif region.distance == PROXIMAL and not region.direction:
+                return "to"
+            elif region.distance == INTERIOR:
```

Comment:
The `in` and `over` branches seem to lack test coverage?

Reply:
Updated to include an `over` branch. `in` already has a unit test case which is succeeding, so I'm confused why that branch isn't covered?

Reply:
Aha - because it is redundant - we already check for `INTERIOR` on line 600.